lib/git-hooks.js

```diff
@@ -111,6 +117,22 @@ module.exports = {
                 return path.resolve(hooksDirname, hookName);
             });
 
+            var ignoreList = [];
```

Comment:
First of all, please move this code to the separate function. Let's call it `exludeIgnoredPathes`, for example.
```
/**
  * @param {String[]} hooks List of hook names to execute.
  * @returns {String[]} hooksWithoutIgnoredPathes
  **/ 
function exludeIgnoredPathes(hooks) {
}
```